docs/GettingStartedOnLinux.md

```diff
@@ -0,0 +1,141 @@
+$ cd flow
+$ sudo mv flow /usr/local/bin/flow
+```
+
+__NOTE__: You can put the `flow` executable anywhere on your PATH. It doesn't have to be in `/usr/local/bin`.  (for advanced users)
+
+#### Installing React Native
+
+OK, with all of that out of the way, it's finally time to install React Native.
+
+For most users of this guide, you'll need to use `sudo` to install React Native as root:
+
+
+```sh
+$ sudo npm install -g react-native-cli
+``
+
+For users who installed NPM through other means, you'll likely want to install React Native as a normal user:
+
+```sh
+$ npm install -g react-native-cli
+```
+
+## Setting up an Android Device
+
+Let's set up an Android device to run our starter project. 
+
+First thing is to plug in your device and check the manufacturer code by using `lsusb`, which should output something like this:
+
+```bash
+$ lsusb
+Bus 002 Device 002: ID 8087:0024 Intel Corp. Integrated Rate Matching Hub
+Bus 002 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+Bus 001 Device 003: ID 22b8:2e76 Motorola PCS 
+Bus 001 Device 002: ID 8087:0024 Intel Corp. Integrated Rate Matching Hub
+Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
+Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+```
+These lines represent the USB devices currently connected to your machine.
+
+You want the line that represents your phone. If you're in doubt, try unplugging your phone and running the command again:
+
+```bash
+$ lsusb
+Bus 002 Device 002: ID 8087:0024 Intel Corp. Integrated Rate Matching Hub
+Bus 002 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+Bus 001 Device 002: ID 8087:0024 Intel Corp. Integrated Rate Matching Hub
+Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
+Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+```
+You'll see that after removing the phone, the line which has the phone model ("Motorola PCS" in this case) disappeared from the list. This is the line that we care about.
+
+`Bus 001 Device 003: ID 22b8:2e76 Motorola PCS`
+
+From the above line, you want to grab the first four digits from the device ID:
+
+`22b8:2e76` 
+
+In this case, it's `22b8`. That's the identifier for Motorola.
+
+You'll need to input this into your udev rules in order to get up and running:
+
+```sh
+$ echo SUBSYSTEM=="usb", ATTR{idVendor}=="22b8", MODE="0666", GROUP="plugdev" | sudo tee /etc/udev/rules.d/51-android-usb.rules
+```
+
+Make sure that you replace `22b8` with the identifier you get in the above command.
+
+Now check that your device is properly connecting to ADB, the Android Debug Bridge, by using `adb devices`.
+
+```bash
+$ adb devices
+List of devices attached
+TA9300GLMK	device
+
+$
+```
+
+For more information, please see the docs for [running an Android app on your device](/react-native/docs/running-on-device-android.html).
```

Comment:
Not the UDEV part. That's completely not covered and needs to be. `adb devices` verifies that UDEV worked.